### ISSUE TITLE
Syntax highlighting for '@import' rules is broken

### ISSUE THREAD

**Issue by spences10** [styled-components/vscode-styled-components#51]
Hi, not sure if this issue is related to [this] but it appears that the colouring is now broken for `injectGlobal`

![image](https://user-images.githubusercontent.com/234708/36028285-4d56c81a-0d96-11e8-9d65-6ac3c9ad15a0.png)

It seems fine with normal components however:

![image](https://user-images.githubusercontent.com/234708/36028325-77c510ca-0d96-11e8-89bb-9eb6eec3d533.png)

Thanks for this really nice extension 
[this]: https://github.com/styled-components/vscode-styled-components/issues/50

**Comment by Briggybros** [comment]
This is not an issue with the parsing of injectGlobal, but instead an error parsing the string in `@import url('{url}');` where the parser sees the closing apostrophe as closing the entire template literal.

**Comment by spences10** [comment]
Ok, so I have worked around it with setting the `import`s url as a var and pass that.

Is that the recommended way to do it now?

![image](https://user-images.githubusercontent.com/234708/36143752-e3c2f174-10a3-11e8-951a-14d1a86d0685.png)

**Comment by Briggybros** [comment]
I think you would have to use `const importUrl = '\'https://fonts.googleapis.com/css?family=Open+Sans|Roboto\'';`

I don't think it should be the _recommended_ way, but it works for now.

**Comment by Briggybros** [comment]
I realise the reason why the parser fails is not as I stated above. There is no parsing rule for @import syntax anymore (was removed by @Cu3PO42 in 6d15d12). The parser breaks when it hits a colon in the url. Removing the http(s):// from the url will 'fix' the breaking of the linting beyond the import rule. However the statement will not be properly highlighted. This 'fix' will break if the url you point to does not support the protocol you are using.
